src/libLookup/Lookup.cpp

```diff
@@ -161,8 +161,8 @@ void Lookup::SetLookupNodes() {
                     }) != m_lookupNodes.end()) {
           continue;
         }
-        // check for hostname only for lookup nodes
-        if (lookupType == "node.lookups") {
+        // check for hostname 
+        if (lookupType == "node.lookups" || lookupType == "node.multiplier") {
```

Comment:
Is it "node.multiplier" or "node.multipliers"?  A few lines later below it is the latter.

Reply:
Had pushed the fix already :)